pcdsdevices/signal.py

```diff
@@ -174,7 +174,6 @@ def _update_avg(self, *args, value, **kwargs):
         with self._lock:
             self.values[self.index] = value
             self.index += 1
-            if self.index == len(self.values):
-                self.index = 0
+            self.index = (self.index + 1) % len(self.values)            
```

Comment:
Trailing whitespace:
```suggestion
            self.index = (self.index + 1) % len(self.values)
```